esx_service/vmdk_ops.py

```diff
@@ -749,9 +749,11 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
                                   tenant_uuid=tenant_uuid,
                                   datastore=datastore)
         elif cmd == "attach":
-            response = attachVMDK(vmdk_path, vm_uuid)
+            with lockManager.get_lock(vm_uuid):
+                response = attachVMDK(vmdk_path, vm_uuid)
         elif cmd == "detach":
-            response = detachVMDK(vmdk_path, vm_uuid)
+            with lockManager.get_lock(vm_uuid):
+                response = detachVMDK(vmdk_path, vm_uuid)
```

Comment:
The client plugin also has a lock (per docker-host), do we need one more lock in the ESX service? The disk level locks are good?